docs/how-to-guides/format-api/README.md

```diff
@@ -1,4 +1,4 @@
-# Introduction to the Format API
+# Formatting Toolbar
```

Comment:
I think it might be worth it to add "API" at the end, as the first paragraph does mention the Format API directly.

Reply:
👍 Added.